extensions/common/iam/identity-trust/identity-trust-sts/identity-trust-sts-embedded/src/main/java/org/eclipse/edc/iam/identitytrust/sts/embedded/SelfIssuedTokenDecorator.java

```diff
@@ -45,7 +44,7 @@ class SelfIssuedTokenDecorator implements JwtDecorator {
     @Override
     public Map<String, Object> claims() {
         var claims = new HashMap<String, Object>(this.claims);
-        claims.put(ISSUED_AT, Date.from(clock.instant()));
+        claims.put("iat", Date.from(clock.instant()));
```

Comment:
Shouldnt this remain a constant?

Reply:
absolutely. IDK why that changed -> using `JWTClaimNames` now.